naurffxiv/src/app/uwu.js

```diff
@@ -1,5 +0,0 @@
-export default function DisplayUWU() {
-	return (
-		<p>Placeholder: Put uwu resources here</p>
-	);
-}
```

Comment:
let's also restore this content since this serves as a template

Reply:
I'm going to murder you in a video game